### DISCUSSION TITLE
Q: List guests and disconnect them

### DISCUSSION THREAD

**Discussion by white-rabbit07** [Art-of-WiFi/UniFi-API-client#153] [Q&A]
Hi. I tried script "list_connected_users.php" to see who's connected ... the output is in JSON ... is it also possible to get the output directly as list?

The other question is about disconnecting any guest. Is there a way to disconnect all existing connections at once?
Thanks!

**Comment by malle-pietje** [comment]
Yes, but that's a matter of adding your own PHP code. Here's rough example:
```php
$data = $unifi_connection->list_sites();

/**
 * provide feedback in json format
 */
//echo json_encode($data, JSON_PRETTY_PRINT);

/**
 * or instead of the above we print each site name and site id
 */
foreach ($data as $site) {
    echo 'Site name: ' . $site->desc . ', site id: ' . $site->name . PHP_EOL;
}
```
There are a lot of good resources to get you started with some basic PHP programming skills, I'm afraid I can't help you out with that.
